installing/installing-mirroring-installation-images.adoc

```diff
@@ -0,0 +1,84 @@
+[id="installing-mirroring-installation-images"]
```

Comment:
Yeah not important for this PR, just more for the future. There's also sometimes an issue if you do a `git mv` and then run an `asciibinder build` before committing, it'll undo the move and make it look like deleted/new files instead :(

So usually if I have to move a file I'll do the `git mv` and then commit right away before running a build to avoid that issue.

Reply:
That is a crazy thing!
I got the `mv` to work. It probably wasn't worth it. There's probably a better way to do it, but this worked:

. On one temporary branch, `git mv` the file and make the updates to it. (I saved a copy of it locally and used DiffMerge.)
. On a second temporary branch, `git cherry-pick` the commit from the PR's branch.
. On the second temporary branch, delete the moved file and amend the commit.
. Cherrypick the commit from the second temporary branch to the first temporary branch.
. Push the first temporary branch and make sure that GitHub recognizes both the renamed file and the rest of the changes from the PR.
. Squash the commits on the first temporary branch, force push, and make sure that GitHub is still content.
. Delete the PR's branch locally.
. Make a new local branch from master with the same name as the PR's branch, cherrypick the commit from the first temporary branch to it, and force push.

Reply:
Ha well I'm glad you worked it out but yeah that's probably more work than it was worth! At that point I probably would have just started the branch fresh and copied the changes over :smile:

Reply:
Believe it or not, that was one of the things that I tried that didn't work! :D

Reply:
Huh that's odd. Oh well :)